src/demo/demo-state.ts

```diff
@@ -52,6 +54,22 @@ export const settSykmeldtMedArbeidsgiver = (value: string) => {
     settILocalStorage(DemoData.SYKMELDT_MED_ARBEIDSGIVER, value);
 };
 
+export const hentRegistreringType = (): string => {
+    return hentFraLocalStorage(DemoData.REGISTRERING_TYPE) || 'INGEN_VERDI';
+};
+
+export const settRegistreringType = (value: string) => {
+    settILocalStorage(DemoData.REGISTRERING_TYPE, value);
+};
+
+export const hentGeografiskTilknytning = (): string => {
```

Comment:
Er hent og sett for geografisk tilknytning i bruk?

Reply:
ikke i denne iterasjonen, men kommer i neste :-)

Reply:
neste er her :-)